packages/agoric-cli/src/commands/auction.js

```diff
@@ -150,7 +150,7 @@ export const makeAuctionCommand = (
           path: { paramPath: { key: 'governedParams' } },
         };
 
-        /** @type {import('@agoric/smart-wallet/src/offers.js').OfferSpec} */
+        /** @type {import('@agoric/smart-wallet/src/smartWallet.js').OfferSpec} */
```

Comment:
if we're going to touch all these lines, maybe it's best to move the type into a `types.js` where it can stay. This is part of the external API of the package.

Reply:
Can we have both types.js and types.d.ts in the same directory, or should they go into types.d.ts?

Reply:
`foo.d.ts` defines the types of a `foo.js`. the importer doesn't know what the syntax is; it always imports the `.js` extension.

so by `into a `types.js`" I meant into whatever file is expressing the types for `types.js`